internal/cmd/command.go

```diff
@@ -158,19 +172,25 @@ func Execute(cmd *cobra.Command, args []string, cfg *v1.Config) error {
 		cmd.PersistentPostRun = u.Collect
 	}
 
-	err := cmd.Execute()
+	err = cmd.Execute()
 	output.ErrPrint(errors.DisplaySuggestionsMessage(err))
 	u.Error = cliv1.PtrBool(err != nil)
 
+	if err := reportUsage(cmd, cfg, u); err != nil {
+		return err
+	}
```

Comment:
These blocks are unrelated, so let's add some whitespace.
```suggestion
	}
	
```